<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>book_returned</class>
 <widget class="QMainWindow" name="book_returned">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1627</width>
    <height>2015</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Book Returned</string>
  </property>
  <property name="windowIcon">
   <iconset resource="bookholics.qrc">
    <normaloff>:/image/1.png</normaloff>:/image/1.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>881</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>57</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>To</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>57</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>From</string>
     </property>
    </widget>
    <widget class="QCalendarWidget" name="to_date">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>296</width>
       <height>183</height>
      </rect>
     </property>
    </widget>
    <widget class="QCalendarWidget" name="from_date">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>30</y>
       <width>296</width>
       <height>183</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>240</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Member</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_member">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">pading:2px;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Borrow ID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_borrow_id">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
        </property>
        <property name="placeholderText">
         <string>Enter valid Borrow ID</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTableWidget" name="return_deatils">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>440</y>
      <width>881</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Book Id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Book Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Book Author</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Borrow Id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Department</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Class</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>370</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ShowDetails</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="bookholics.qrc"/>
 </resources>
 <connections/>
</ui>
